src/test-utils/dom/flashbar/index.ts

```diff
@@ -16,6 +16,19 @@ export default class FlashbarWrapper extends ComponentWrapper {
     return this.findAllByClassName(styles['flash-list-item']).map(item => new FlashWrapper(item.getElement()));
   }
 
+  /**
+   * Returns the individual flashes of this flashbar given the item type.
+   *
+   * If the items are stacked, only the item at the top of the stack is returned.
+   *
+   * If an item is loading assuming its type is "info".
```

Comment:
this line isn't totally clear, are we saying the test util makes this assumption, or that consumers of the test util should make that assumption?